### ISSUE TITLE
Parent relations can not be identified

### ISSUE THREAD

**Issue by ManDay** [openstreetmap/iD#8313]
Take a feature like a street and look at the associated relations like bus routes. In the feature's properties, there will be a list of relations, whose names are potentially shortened to something like
> **Bus Route** Bus line number 123 from Bal…

From this point, there seems to be no way to identify the actual relation. There is no visible ID or otherwise. Clickin on the relation will select the relation, but still, no unique identification is possible as the properties panel doesn't show anything, either.

This seems to be particularly problematic for these relations because I noticed that in order to make substantial changes to features which are bound to relations, one often has to delete and later reconstruct these relations. I currently drop back into the OSM map view to identify them and match them by name, but not only is this very tedious but also not entirely safe, since I thereby rely on recognizing them by their names (which, is not unique and made more difficult by the shortening).

**Comment by maro-21** [comment]
Can you give examples of relations with the same name? If they are bus routes they shouldn't have the same name.

You know you can always expand the left panel:

![relations](https://user-images.githubusercontent.com/57260466/105632503-aba5d300-5e53-11eb-8ab4-1d8192b475eb.png)
----
![relations2](https://user-images.githubusercontent.com/57260466/105632505-af395a00-5e53-11eb-96e9-7f66e16b1c48.png)

**Comment by ManDay** [comment]
I did not know that the left panel can be resized. I tried to drag it by the very edge but only now discovered that you have to drag besides the edge, outside of it. My remark about coinciding names was more to illustrate the point of my having difficulties identifying them.

Maybe I am going about this wrong way, but even *with* the enlarged panel *and* unique names I am still having difficulties. I need to split a road to turn it into a double carriage way lke this
```
___                          ___
   \___   = (becomes) =>        \____
___/                         ___/‾‾‾‾  
```
but iD won't let me perform the split until I have deleted a dozen bus route relations on the road segment, so I need to restore them later. Some of the bus routes had very similar names and were also written in a script I'm not fluent in and to which I have no keyboard. That's where comparing relations by name became difficult for me.

So what I did is go into the OSM interface and copy their node-IDs from there. It was good to find that, when recreating the relations on the road segment(s), iD would accept the node-IDs as input and auto-complete to the correct relation (by name), so that direction of putting IDs into the relation field works fine. Just obtaining the ID in the first place was not possible inside of iD.

**Comment by maro-21** [comment]
> but iD won't let me perform the split until I have deleted a dozen bus route relations on the road segment, so I need to restore them later

Yes, this is unfortunately a disadvantage. It has to be done manually...

**Comment by 1ec5** [comment]
For consistency, a tooltip should be added to the label that contains the relation name, so that you can see the full name regardless of the sidebar’s width. There’s already a similar tooltip in the dropdown menu of nearby relations when adding a feature to a relation.

**Comment by matkoniecz** [comment]
Can you give an example of location where it is a problem after latest round of relation list improvements?

**Comment by tyrasd** [comment]
True, I believe this has improved significantly in the meantime, so I guess it's fine now

**Comment by matkoniecz** [comment]
I bet that it is still problem somewhere, but we need specific example to act on.
